core-libs/setup/ssr/error-handling/server-errors/cx-errors.ts

```diff
@@ -11,6 +11,6 @@
  *  - CmsPageNotFoundServerError SR0002: Used when the requested CMS page is not found.
  * */
 export enum CxError {
-  UnknownServerError = 'SR0001',
-  CmsPageNotFoundServerError = 'SR0002',
+  UnknownServerError = 'UnknownServerError',
+  CmsPageNotFoundServerError = 'CmsPageNotFoundServerError',
 }
```

Comment:
I'm sorry, I still need a help to understand why we need this enum.

btw. Those values are not about Spartacus _errors_, but rather `ServerErrorResponse`s, so the JSDoc and the enum name are also a bit confusing to me.

Reply:
As we discussed, let's abandon the enum for now and use hardcoded values inside server error response classes.